index.js

```diff
@@ -3,109 +3,12 @@ const hamburger = document.querySelector('.hamburger');
 const closeIcon = document.querySelector('.closeIcon');
 const menuIcon = document.querySelector('.menuIcon');
 
-function toggleMenu() {
-    if (menu.classList.contains('showMenu')) {
-        menu.classList.remove('showMenu');
-        closeIcon.style.display = 'none';
-        menuIcon.style.display = 'block';
-    } else {
-        menu.classList.add('showMenu');
-        closeIcon.style.display = 'block';
-        menuIcon.style.display = 'none';
-    }
-}
+const toggleMenu = () => {
+  const showMenu = menu.classList.toggle('showMenu');
+  closeIcon.style.display = showMenu ? 'block' : 'none';
+  menuIcon.style.display = showMenu ? 'none' : 'block';
+};
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=mobile%20menu%20disappears.-,When%20the%20user%20clicks%20(or%20taps)%20any%20of%20the%20mobile%20menu%20options%2C%20a%20correct%20part%20of%20the%20page%20is%20displayed.,-We%20included%20a), When I click `contact` in mobile menu option it doesn't display the contact form section, please fix that.